### ISSUE TITLE
Import from other Shlink instance fails if a link has no visits

### ISSUE THREAD

**Issue by loelkes** [shlinkio/shlink#1135]
#### How Shlink is set-up

* Shlink Version: 2.7.2
* PHP Version: 8.0.6
* How do you serve Shlink: Docker image
* Database engine used: MariaDB

I use `cli short-url:import` short URLs with visits from another instance. The import fails at links that have 0 visits. Clicking on that link and therefore setting the counter to 1 solves this.

```
/etc/shlink/bin # ./cli short-url:import

 What is the source you want to import from::
  [0] bitly
  [1] csv
  [2] shlink
 > 2


 What is your Shlink instance base URL?:
 > ...

 What is your Shlink instance API key?:
 > ...

 Do you want to import each short URL's visits too? (yes/no) [yes]:
 > yes

 https://...: Imported with 176 visits
 https://...: Imported with 34 visits
 https://...: with 1 visits

In InvalidRequestException.php line 28:
                                                                                                                    
  Request to https://.../rest/v2/short-urls/.../visits?page=0&itemsPerPage=300 failed with status code 500  
```

**Comment by acelaya** [comment]
Apparently there are at least two bugs here.

First, the import should not try to load visits when the short URL has none https://github.com/shlinkio/shlink-importer/issues/32

However, this only has the side effect that it makes extra API requests. The big issue seems to be that Shlink crashes when trying to load visits for page 0. Instead, it should probably fall back to page 1 if the value is lower than 1.

And I could even find a third bug. The import should not crash completely if one call fails. It should skip one short URL with a warning, but continue with the rest. https://github.com/shlinkio/shlink-importer/issues/new

**Comment by acelaya** [comment]
I have just released v2.7.3, which addresses these issues.

I have finally left the behavior of failing as a whole as it was. It's hard to predict all kinds of possible issues in order to properly handle them, so better fix the known issues, and leave a transactional behavior for any other case.

If you are using the docker image, you will have to wait for this job to finish (~1h) https://github.com/shlinkio/shlink/runs/3223813699?check_suite_focus=true
